Gems/WhiteBox/Code/Source/EditorWhiteBoxComponentModeTypes.h

```diff
@@ -16,6 +16,12 @@ namespace AZ
     class Color;
 }
 
+namespace AzToolsFramework 
+{
+    struct ActionOverride;
+} // namespace AzToolsFramework
+
+
```

Comment:
uber nit: Please remove the additional line break, in the White Box Gem we aim for 1 max line breaks between code segments, no more